src/nav/ModalNavBar.js

```diff
@@ -76,11 +78,4 @@ class ModalNavBar extends PureComponent<Props> {
   }
 }
 
-export default connect(
-  (state: GlobalState) => ({
-    canGoBack: getCanGoBack(state),
-  }),
-  null,
-  null,
-  connectPreserveOnBackOption,
-)(ModalNavBar);
+export default connect((state: GlobalState) => ({}), null, null)(ModalNavBar);
```

Comment:
`(state: GlobalState) => ({}), null, null)` is that part even needed?
If there is no `dispatch` even the `connect` is not needed.

Reply:
What I meant was that you don't need to pass the three empty parameters.
```js
export default connect((state: GlobalState) => ({}), null, null)(ModalNavBar);
```
can become
```js
export default connect()(ModalNavBar);
```

Reply:
Cool! got it 👍